### ISSUE TITLE
MD034 false positive for HTML link with line break

### ISSUE THREAD

**Issue by scottamain** [DavidAnson/markdownlint#827]
With `no-inline-html: false` (MD033), I get a false positive for `no-bare-urls` (MD034) if there's a line break in the HTML tag. For example, if the HTML looks like this then MD034 flags it:

```
<a class="button" href="https://www.example.com"
>Sign up</a>
```

But this is okay:

```
<a class="button" href="https://www.example.com">Sign up</a>
```

**Comment by scottamain** [comment]
That's interesting! Okay, although it's inconsistent with browser behavior, I can understand why MD parsers are more particular about where newlines appear inside a tag (it works if I put the newline _between_ tag attributes). Thank you for the quick response!
